README-PL.md

```diff
@@ -150,10 +150,9 @@ Linki wyszukiwania które wskazują na kwestie skierowane dla początkujących.
 
 ## Inicjatywy kontrybucji w otwartym oprogramowaniu
 - [Up For Grabs](https://up-for-grabs.net/) - Zawiera projekty z zagadnieniami dla początkujących.
-- [First Timers Only](https://www.firsttimersonly.com/) - Lista błędów oznaczonych dla tych co dopiero zaczynają *(ang. first timers only)*
```

Comment:
This can be kept and added back in.

Reply:
add66b602f87077fb9a28bfc9c24e09203be26b5 done